hnet/constants.go

```diff
@@ -7,6 +7,7 @@ const (
 	CLIENT_REQUEST_STATS       uint32 = 11
 	CLIENT_START_SPECTATING    uint32 = 14
 	CLIENT_STOP_SPECTATING     uint32 = 15
+	CLIENT_HAS_MAP             uint32 = 16
```

Comment:
I would probably name it something like `CLIENT_SPECTATOR_HAS_MAP` & `SERVER_SPECTATOR_HAS_MAP`

Reply:
Done